git-tar/function/handler.go

```diff
@@ -296,3 +305,22 @@ func findStackFile(repo *sdk.PushEventRepository) (bool, error) {
 
 	return false, nil
 }
+
+func getRawURL(scm string, repositoryURL string, repositoryOwnerLogin string, repositoryName string) (string, error) {
+
+	addrPrefix := ""
+	rawPath := ""
+	switch scm {
```

Comment:
`scm` stands for source control management - so this should be only `github` or only `gitlab`. Not sure why we are adding the text `-push` here? Please change.

Reply:
These are the values that come from pushEvent.SCM. 
We need to change them in order to omit the `-push` suffix here

Reply:
I added the change but directly with `Source` constant. This can be quickly changed.